packages/snaps-controllers/package.json

```diff
@@ -16,8 +16,9 @@
   ],
   "scripts": {
     "test:prepare": "yarn mkdirp test/fixtures && ./scripts/generate-fixtures.sh",
-    "test": "yarn test:prepare && jest && yarn posttest",
-    "posttest": "jest-it-up --margin 0.25",
+    "test": "shx rm -rf coverage && yarn test:prepare && jest && yarn test:browser && yarn posttest",
```

Comment:
either use `shx rm -rf` everywhere and remove usage of `rimraf` in all package.json, or use `rimraf` instead